jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/ButtonToggle.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.jdi.light.angular.elements.complex;
+
+import com.epam.jdi.light.angular.asserts.TabsAssert;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+
+public class ButtonToggle extends UIBaseElement<TabsAssert> {
```

Comment:
To my mind, when introducing a new element it is necessary to create an appropriate Assert Class.
ButtonToggleAssert, not TabsAssert.

Reply:
ButtonToggle functionality is not checked in these tests. Used as additional element to use TabGroup functioanality